src/V3SplitVar.cpp

```diff
@@ -894,15 +888,16 @@ class PackedVarRef {
     AstBasicDType* m_basicp;  // Cache the ptr since varp->dtypep()->basicp() is expensive
     bool m_dedupDone;
     static void dedupRefs(std::vector<PackedVarRefEntry>& refs) {
-        vl_unordered_map<AstNode*, size_t> nodes;
+        // Use raw pointer to dedup
+        typedef std::map<AstNode*, size_t> NodeIndices;
```

Comment:
This I think can remain an vl_unordered_map.  Or if not (if it needs ordering), then it presumably needs your compare function.

Reply:
Oh, thank you. I dropped the predicate by mistake in 00ff30.
Resurrected in b5ae9be861f6d528a8d2aa5f9e9ac3b1b6f2da48.